### PR TITLE
New package: gitlab-runner-11.8.0

### PR DESCRIPTION
**Pull request by the-wrench99**

### CODE REVIEW

**Comment by Cogitri** [comment]
Oh, does gitlab runner actually work on runit now? Last time I tried it only worked on systemd and sysvinit

File: srcpkgs/gitlab-runner/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Official GitLab CI runner written in Go"
+maintainer="Christian Poulwey <christian.poulwey@t-online.de>"
+license="MIT"
+homepage="https://docs.gitlab.com/runner/"
+distfiles="https://gitlab.com/gitlab-org/gitlab-runner/-/archive/v${version}/gitlab-runner-v${version}.tar.gz"
+checksum=b334e446c996441937b8f2f5b995da2687f9b52c29fb87ba76a9ae4a9614b68a
+
+post_install() {
+	vsv gitlab-runner
+	vlicense ${wrksrc}/LICENSE MIT
```
Comment (CameronNemo) [inline srcpkgs/gitlab-runner/template]: I think you could just use `vlicense LICENSE`, no?

File: srcpkgs/gitlab-runner/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'gitlab-runner'
+pkgname=gitlab-runner
+version=11.8.0
+revision=1
+wrksrc=${pkgname}-v${version}
```
Comment (CameronNemo) [inline srcpkgs/gitlab-runner/template]: please quote

**Comment by the-wrench99** [comment]
Cogitri: The `gitlab-runner start` command forks into background whie the `gitlab-runner run` command does not.
CameronNemo: Done

**Pull request merged**